### PR TITLE
Fix/hck 3232 fix cloud platform name url

### PR DESCRIPTION
**Pull request by mtseluiko**

### CODE REVIEW

File: reverse_engineering/api.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ const connect = async (connectionInfo, logger, cb, app) => {
 	}
 };
 
-const disconnect = async (connectionInfo, logger, cb) => {
+const disconnect = async (connectionInfo, logger, cb, app) => {
```
Comment (mtseluiko) [inline reverse_engineering/api.js:20]: missing app

File: reverse_engineering/helpers/snowflakeHelper.js [hunk 2]
```diff
@@ -28,13 +31,25 @@ const connect = async (logger, { host, username, password, authType, authenticat
...
+		authPromise = authByOkta(logger, { account, accessUrl, username, password, authenticator, role, warehouse, timeout });
 	} if (authType === 'externalbrowser') {
-		return authByExternalBrowser(logger, { account, accessUrl, token, proofKey, username, password, role, warehouse, timeout })
+		authPromise = authByExternalBrowser(logger, { account, accessUrl, token, proofKey, username, password, role, warehouse, timeout })
 	} else {
-		return authByCredentials({ account, username, password, role, warehouse, timeout });
+		authPromise = authByCredentials({ account, username, password, role, warehouse, timeout });
 	}
+
+	return await authPromise.catch(err => {
```
Comment (mtseluiko) [inline reverse_engineering/helpers/snowflakeHelper.js:43]: so, if there is the "can't reach snowflake" issue, I check and try to connect with cloud provider in host name

File: reverse_engineering/helpers/snowflakeHelper.js [hunk 3]
```diff
@@ -346,6 +363,16 @@ const splitEntityNames = names => {
 
 const isView = name => name.slice(-4) === ' (v)';
 
+const getSchemasInfo = async () => {
```
Comment (mtseluiko) [inline reverse_engineering/helpers/snowflakeHelper.js:366]: log the schemas data

File: reverse_engineering/helpers/snowflakeHelper.js [hunk 4]
```diff
@@ -440,7 +467,7 @@ const getDDL = async (tableName, logger) => {
 		return getFirstObjectItem(_.first(queryResult));
 	} catch (err) {
 		logger.log('error', { tableName, message: err.message, stack: err.stack }, 'Getting table DDL')
-		throw err;
+		return ''
```
Comment (mtseluiko) [inline reverse_engineering/helpers/snowflakeHelper.js:470]: if user have no rights to get ddl - we still able to parse table/view and not have errors, but in logs there will be messages about this error

**Review by lenchvolodymyr — APPROVED** [review]

**Pull request merged**